dvc/repo/artifacts.py

```diff
@@ -17,7 +17,7 @@
 from dvc.utils.serialize import modify_yaml
 
 if TYPE_CHECKING:
-    from gto.tag import Tag as GTOTag
+    from gto.tag import Tag as GTOTag  # type: ignore[import-not-found]
```

Comment:
`gto` is a required dependency. This may be due to some environment issues in your machine. These `# type: ignore` should not be necessary.